src/test/java/ParserTest.java

```diff
@@ -0,0 +1,37 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import java.util.ArrayList;
+
+public class ParserTest {
+
+    @Test
+    public void testFind() {
+        try {
+            ArrayList<String> expectedList = new ArrayList<>();
+            expectedList.add("Find");
+            expectedList.add("book");
+            Parser parser = new Parser();
+            assertEquals(expectedList, parser.processString("find book", 10));
+        } catch (Exception e) {
+            fail();
+        }
```

Comment:
Hi, I added the fail so that I will know Duke won't throw exceptions when it is not supposed to. Though I suppose the testing is a bit too 'gentle'? -_-||